addons/phone_validation/lib/phonenumbers_patch/__init__.py

```diff
@@ -40,5 +40,10 @@ def _local_load_region(code):
         # region_PA.py in the current directory was copied from external source:
         # https://github.com/daviddrysdale/python-phonenumbers/blob/v8.12.43/python/phonenumbers/data/region_PA.py
         phonenumbers.phonemetadata.PhoneMetadata.register_region_loader('PA', _local_load_region)
+    # Patching phonemetadata of Senegal if using old phonenumbers version
```

Comment:
Just because I have some TOCs :D 
```suggestion
    # MONKEY PATCHING phonemetadata of Senegal if using old phonenumbers version
```

Reply:
I actually wanted to (optionally) refactor it, so i'm making a second [ref] commit for that